addons/web/static/src/legacy/scss/bootstrap_overridden.scss

```diff
@@ -186,3 +186,19 @@ $code-color: $o-main-code-color!default;
 $kbd-color: $o-gray-700 !default;
 $kbd-bg: $o-gray-100 !default;
 $kbd-box-shadow: 0px 1px 1px rgba($o-black, 0.2), inset 0px -1px 1px 1px rgba($o-gray-200, 0.8), inset 0px 2px 0px 0px rgba($o-white, 0.8) !default;
+
+// Border-radius
+
+$border-radius-lg: .6rem !default;
+
+// Buttons
```

Comment:
I know it's still WIP, just 2 notes:
- This should be done in another PR ;)

- `$border-radius-lg: .6rem !default;` Rather than set an arbitrary value, use a custom variable defined in `primary_variables.scss`. You may have noticed that we already did it in enterprise; to use the same naming/logic would be cleaner.

Reply:
> This should be done in another PR ;)

Indeed ! And it’s already here : https://github.com/odoo/odoo/pull/87521  (I had not yet written it in the description 🙂 )

> $border-radius-lg: .6rem !default; Rather than set an arbitrary value, use a custom variable defined in primary_variables.scss. You may have noticed that we already did it in enterprise; to use the same naming/logic would be cleaner.

I updated the code 👍